### PR TITLE
Disable AD join os check option

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Even if the server is already joined to a domain (from the system) this option permit to bypass the check and allow the join through the admin gui.

# Impacts
AD Join

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* Disable AD join os check option

### CODE REVIEW

File: conf/documentation.conf [hunk 1]
```diff
@@ -1204,6 +1204,13 @@ The algorithm used to calculate the VLAN in a VLAN pool.
 EOT
 guide_anchor=_introduction_to_role_based_access_control
 
+[advanced.active_directory_os_join_check_bypass]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Ability to disable the check if the os has been join to a domain.
```
Comment (extrafu) [inline conf/documentation.conf]: I find this confusing... we enable to disable the check?
Perhaps rephrase it like this:

Enable to bypass the operating system domain join verification.

File: conf/pf.conf.defaults [hunk 2]
```diff
@@ -871,7 +871,11 @@ multihost=disabled
 #
 # The algorithm used to calculate the VLAN in a VLAN pool.
 vlan_pool_technique = username_hash
-
+#
+# active_directory_os_join_check_bypass
+#
+# Ability to disable the check if the os has been join to a domain.
+active_directory_os_join_check_bypass=disable
```
Comment (extrafu) [inline conf/pf.conf.defaults]: disabled

File: html/pfappserver/lib/pfappserver/I18N/en.po [hunk 3]
```diff
@@ -9812,6 +9812,10 @@ msgstr "Update the iplog using the accounting"
 msgid "advanced.vlan_pool_technique"
 msgstr "VLAN pool technique"
 
+# conf/documentation.conf
+msgid "advanced.active_directory_os_join_check_bypass"
+msgstr "Disable AS AD join"
```
Comment (extrafu) [inline html/pfappserver/lib/pfappserver/I18N/en.po]: AS?

File: conf/pf.conf.defaults [hunk 4]
```diff
@@ -871,7 +871,11 @@ multihost=disabled
 #
 # The algorithm used to calculate the VLAN in a VLAN pool.
 vlan_pool_technique = username_hash
-
+#
+# active_directory_os_join_check_bypass
```
Comment (extrafu) [inline conf/pf.conf.defaults:875]: Same as above

File: html/pfappserver/lib/pfappserver/I18N/en.po (hunk 3 above)
Comment (fdurand) [reply html/pfappserver/lib/pfappserver/I18N/en.po]: OS

Comment (julsemaan) [reply html/pfappserver/lib/pfappserver/I18N/en.po]: I think he meant ASS AD join

**Comment by fdurand** [comment]
Bump

**Pull request merged**